crates/pop-cli/src/commands/mod.rs

```diff
@@ -45,6 +46,9 @@ pub(crate) enum Command {
 	/// Remove generated/cached artifacts.
 	#[clap(alias = "C")]
 	Clean(clean::CleanArgs),
+	/// Add a new feature to your existing polkadot-sdk project
+	#[clap(name = "add-to", alias = "a")]
```

Comment:
I feel like `pop add-to` does not look very good as a CLI subcommand so proposes a few suggestions below

```suggestion
	#[clap(name = "insert", alias = "a")]
```

or 

```suggestion
	#[clap(name = "expand", alias = "a")]
```

or maybe just 

```suggestion
	#[clap(name = "add", alias = "a")]
```

Reply:
Hey Tin, the command was originally called `pop add` as you mentioned. As a standalone command, it looks much better than `pop add-to`, the issue here comes when you actually use it. `pop add runtime pallet` may feel a bit ambiguous IMO, what are you adding? a pallet **to** a runtime? a runtime and a pallet to the workspace? 

That's why I came up with the name `pop add-to` as I believe it emphasizes better what it does: `pop add-to runtime pallet` -> Add a pallet to a runtime. 

But maybe I'm overthinking 😅, just let me know what do you think about this, we can easily change this name if you still consider other names fit better

Reply:
Hmm I see. What do you think about `expand`? Expanding a runtime with a pallet, expanding a pallet with a config type. 

```
pop expand runtime pallet
```

```
pop expand pallet config-type
```